tests/test_coverity.py

```diff
@@ -75,4 +89,8 @@ def test_code_quality_with_config(self):
             str(TEST_IN_DIR / 'defects.txt'),
         ])
         self.assertEqual(3, retval)
-        self.assertTrue(filecmp.cmp(out_file, ref_file))
+        with open(out_file) as file:
+            cq_out = json.load(file)
+        with open(ref_file) as file:
+            cq_ref = json.load(file)
+        self.assertEqual(ordered(cq_out), ordered(cq_ref))

```

Comment:
What is the reason for replacing `filecmp.cmp`?

Reply:
`filecmp.cmp` takes the order into account. But the order depends on the order the checkers of the different classifications are saved, and thus can be different. Therefore, I ordered them before comparing.

Reply:
Resolved in 45fcc8a74499d5f924acf050bcc2bed24534fac3